spec/user_spec.rb

```diff
@@ -0,0 +1,209 @@
+require "spec_helper"
+
+describe User do
```

Comment:
`spec/support/clearance.rb` - updated descriptions to make it clearer! :rainbow: